webpack.common.cjs

```diff
@@ -0,0 +1,23 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'index.js'
+  },
+  node: false,
+  target: 'node',
+  plugins: [
+    new webpack.optimize.LimitChunkCountPlugin({
+      maxChunks: 1,
+    })
+  ],
+  externals: {
+    http: 'http',
+    https: 'https',
+    deepmerge: 'deepmerge',
+    'base64-arraybuffer': 'base64-arraybuffer'
```

Comment:
Externals need to be marked as CommonJS otherwise Webpack will for the deps on the global object. `commonjs2` is for libraries that export onto `exports.default`.
```suggestion
    http: { commonjs: 'http' },
    https: { commonjs: 'https' },
    deepmerge: { commonjs2: 'deepmerge' },
    'base64-arraybuffer': { commonjs2: 'base64-arraybuffer' }
```